frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -289,6 +289,7 @@ ACTION_TYPE_FAILED_ILLEGAL_OS_TYPE_IS_NOT_SUPPORTED_BY_ARCHITECTURE_TYPE=Cannot
 ACTION_TYPE_FAILED_ILLEGAL_VM_DISPLAY_TYPE_IS_NOT_SUPPORTED_BY_OS=Cannot ${action} ${type}. Selected display type is not supported by the operating system.
 ACTION_TYPE_FAILED_ILLEGAL_VM_DISPLAY_TYPE_IS_NOT_SUPPORTED_BY_FIRMWARE=Cannot ${action} ${type}. Selected display type is not supported by the firmware.
 ACTION_TYPE_FAILED_ILLEGAL_WATCHDOG_MODEL_IS_NOT_SUPPORTED_BY_OS=Cannot ${action} ${type}. Selected watchdog model is not supported by the operating system.
+ACTION_TYPE_FAILED_DISK_ALREADY_EXISTS=Cannot ${action} ${type}. One of the Template Disks already exists.
```

Comment:
Its not a Template disk. But just a disk.

` ACTION_TYPE_FAILED_DISK_ALREADY_EXISTS=Cannot ${action} ${type}. One of the disks already exists.`